surveys/survey-validator/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: survey_validator
+description: A sample command-line application.
```

Comment:
Consider adding a short description here for people visiting this repository in the future.